pkg/nodes/enricher/enricher.go

```diff
@@ -1,18 +1,20 @@
 package enricher
 
 import (
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
 	"github.com/stackrox/rox/generated/storage"
 	pkgMetrics "github.com/stackrox/rox/pkg/metrics"
 	"github.com/stackrox/rox/pkg/scanners"
 	"github.com/stackrox/rox/pkg/scanners/clairify"
+	"github.com/stackrox/rox/pkg/scanners/scannerv4"
 	"github.com/stackrox/rox/pkg/scanners/types"
 )
 
 // NodeEnricher provides functions for enriching nodes with vulnerability data.
 //
 //go:generate mockgen-wrapper
 type NodeEnricher interface {
-	EnrichNodeWithInventory(node *storage.Node, nodeInventory *storage.NodeInventory) error
+	EnrichNodeWithInventory(node *storage.Node, nodeInventory *storage.NodeInventory, indexReport *v4.IndexReport) error
```

Comment:
If we keep that name, we may need to add a comment here to explain the confusion with NodeInventory in the name and IndexReport in the parameters.